language/tests/regent/run_pass/import_cross_product.rg

```diff
@@ -0,0 +1,52 @@
+-- Copyright 2021 Stanford University
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+import "regent"
+
+local c = regentlib.c
+local fmt = require('std/format')
+
+terra bar(p : c.legion_logical_partition_t,
```

Comment:
Is there any reason this can't be a `__demand(__inline) task`?

Reply:
Ok, I've manually inlined it for now. Let me know if you want to change it.